scripts/dev-integration-tests.sh

```diff
@@ -44,6 +44,12 @@ then
     echo "Please build the image before running integration tests"
     exit 1
 fi
+if [ "x$(docker images|grep ligato/networkservicemesh/sidecar-injector)" == "x" ]
```

Comment:
This should be `networkservicemesh/sidecar-injector`.

Reply:
Thanks will fix it.